new_lineage_panel/src/App.tsx

```diff
@@ -172,79 +183,54 @@ function App() {
       aiEnabled: boolean;
     }) => {
       setShowSidebar(false);
-      if (!args) {
-        return;
-      }
-      const { node, aiEnabled: _aiEnabled } = args;
-      aiEnabled = _aiEnabled;
+      if (!args) return;
+      aiEnabled = args.aiEnabled;
```

Comment:
should `aiEnabled` static variable? can this be set as ref or state variable? this type of variable declarations go against react patterns

Reply:
this should be fine as it is mostly constant through out the app